src/components/Home/index.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react'
 import Wrapper from '@components/Wrapper'
-import ProjectsHeader from '@components/ProjectsHeader'
-import ProjectList from '@components/ProjectList'
+import ProjectsStatsContainer from '@containers/ProjectsStatsContainer'
+import ProjectListContainer from '@containers/ProjectListContainer'
 
 const Home = () => (
   <Wrapper flex>
     <Wrapper center width="regular" mTop="large">
-      <ProjectsHeader />
-      <ProjectList />
+      <ProjectsStatsContainer />
+      <ProjectListContainer />
```

Comment:
I know we discussed it before but in such cases if the two containers are siblings in the same component it might be worth it to have a container for the Homepage and pass down the data directly from here.
That way you can handle:
- Loading state
- Empty state
- (Possibly) errors too

Reply:
Since we use the `ProjectsStats` in the project single page as well, we should simply have a container for that page.